src/java/org/apache/cassandra/auth/IAuthorizer.java

```diff
@@ -62,12 +62,13 @@ default boolean requireAuthorization()
      * @param permissions Set of permissions to grant.
      * @param resource Resource on which to grant the permissions.
      * @param grantee Role to which the permissions are to be granted.
+     * @return the permissions that have been successfully granted.
```

Comment:
We might mention that this is different to the `permissions` attribute, for example:
```suggestion
     * @return the permissions that have been successfully granted, comprised by the requested permissions excluding
     * those permissions that were already granted.
```